mypartners/views.py

```
@@ -1202,18 +1220,23 @@ def process_email(request):
 
     validated_contacts = []
     for element in contact_emails:
-        if not element.lower() == 'prm@my.jobs' and validate_email(element):
+        if not element.lower() == PRM_EMAIL and validate_email(element):
             validated_contacts.append(element)
 
     contact_emails = validated_contacts
 
     try:
         contact_emails.remove(admin_email)
     except ValueError:
+        newrelic.agent.record_exception(*sys.exc_info())
```

I just removed them. Not convined new relic is buying us anything here. Especially if it's a silent failure not caused by an exception. In any case, the file logging still remains.